tests/forms/test_address_upload.py

```diff
@@ -0,0 +1,170 @@
+import pytest
+
+from pretix.base.models import (
+    User, Team,
+)
+from django.core.files.uploadedfile import SimpleUploadedFile
+from pretix_eth.models import WalletAddress
+
+
+@pytest.fixture
+def env(event):
+    user = User.objects.create_user('admin@pretix', 'admin')
```

Comment:
two things.

1. I'm *assuming* that the first argument here is an email address?  I find it nice to use keyword arguments in cases like this to document what the fields are.
2. If the first is an email, I would recommend using `@example.com` email addresses as a best practice.